### PR TITLE
Support only news flashes from the supported location categories list…

### PR DESCRIPTION
**Pull request by ziv17**
… in news_flash API and cache insertion.

Rename anyway/dataclasses to anyway/anyway_dataclasses.

### CODE REVIEW

File: anyway/views/news_flash/api.py [hunk 1]
```diff
@@ -25,14 +51,14 @@ def news_flash():
     road_segment = request.values.get("road_segment_only")
     offset = request.values.get("offset", DEFAULT_OFFSET_REQ_PARAMETER)
     limit = request.values.get("limit", DEFAULT_LIMIT_REQ_PARAMETER)
-    news_flash_obj = db.session.query(NewsFlash)
-    news_flash_obj.offset(offset)
+    query = session.query(NewsFlash)
+    query.offset(offset)
```
Comment (BarVolunteering) [inline anyway/views/news_flash/api.py:55]: Did you mean `query = query.offset(offset)`?
*I think @atalyaalon has touched this code in her recent pull request

File: static/js/app_send_email.min.js [hunk 2]
```diff
@@ -1 +0,0 @@
...2wuH2SnBzbZAbPJwpPx+VQuzcm7dgRCB57a1uBzUDRL4bfnI0RE0eaXd9W89mpjqHZnUI5Hh2l2dkZZUhOqpi2qSmpOmZ64Tuu9qlz/SEXo6MEHa3wOip46F1n7633eekV8ds8Wxjn37Wl63VVa+ej5oeEZ/82ZBETJjpJ1Rbij2D3Z/1trXUvLsblCK0XfOx0SX2kMsn9dX+d+7Kf6h8o4AIykuffjT8L20LU+w4AZd5VvEPY+XpWqLV327HR7DzXuDnD8r+ovkBehJ8i+y8YAAAAASUVORK5CYII=)"},warn:{color:"#C09853","background-color":"#FCF8E3","border-color":"#FBEED5","background-image":"url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAMAAAC6V+0/AAABJlBMVEXr6eb/2oD/wi7/xjr/0mP/ykf/tQD/vBj/3o7/uQ//vyL/twebhgD/4pzX1K3z8e349vK6tHCilCWbiQymn0jGworr6dXQza3HxcKkn1vWvV/5uRfk4dXZ1bD18+/52YebiAmyr5S9mhCzrWq5t6ufjRH54aLs0oS+qD751XqPhAybhwXsujG3sm+Zk0PTwG6Shg+PhhObhwOPgQL4zV2nlyrf27uLfgCPhRHu7OmLgAafkyiWkD3l49ibiAfTs0C+lgCniwD4sgDJxqOilzDWowWFfAH08uebig6qpFHBvH/aw26FfQTQzsvy8OyEfz20r3jAvaKbhgG9q0nc2LbZxXanoUu/u5WSggCtp1anpJKdmFz/zlX/1nGJiYmuq5Dx7+sAAADoPUZSAAAAAXRSTlMAQObYZgAAAAFiS0dEAIgFHUgAAAAJcEhZcwAACxMAAAsTAQCanBgAAAAHdElNRQfdBgUBGhh4aah5AAAAlklEQVQY02NgoBIIE8EUcwn1FkIXM1Tj5dDUQhPU502Mi7XXQxGz5uVIjGOJUUUW81HnYEyMi2HVcUOICQZzMMYmxrEyMylJwgUt5BljWRLjmJm4pI1hYp5SQLGYxDgmLnZOVxuooClIDKgXKMbN5ggV1ACLJcaBxNgcoiGCBiZwdWxOETBDrTyEFey0jYJ4eHjMGWgEAIpRFRCUt08qAAAAAElFTkSuQmCC)"}}});
```
Comment (BarVolunteering) [inline static/js/app_send_email.min.js:1]: Why did you deleted this file?

File: anyway/oauth.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 from flask import current_app, url_for, request, redirect, Response
 from rauth import OAuth2Service
 
-from anyway.dataclasses.user_data import UserData
+from anyway.anyway_dataclasses.user_data import UserData
```
Comment (BarVolunteering) [inline anyway/oauth.py:9]: I changed the folder name in my recent pull request - but this name is also good.

**Review by BarVolunteering — COMMENTED** [review]
Good work!

**Comment by BarVolunteering** [comment]
@ziv17 The tests are failing - as it turnout there is a difference for the order of adding the elements.

File: anyway/views/news_flash/api.py [hunk 4]
```diff
@@ -9,13 +9,39 @@
...
+    query = gen_news_flash_query(db.session)
+    news_flashes = query.all()
+
+    news_flashes_jsons = [n.serialize() for n in news_flashes]
+    for news_flash in news_flashes_jsons:
+        set_display_source(news_flash, news_flash_id)
+    return Response(json.dumps(news_flashes_jsons, default=str), mimetype="application/json")
+
+
+def gen_news_flash_query(session):
```
Comment (BarVolunteering) [inline anyway/views/news_flash/api.py:44]: I am not sure how i missed it - Sometimes the function `gen_news_flash_query` return an object of type `Response` like in line 60, and sometimes an a SQL query object - like in line 106.

Is this indented?
Maybe add type hinting to the function

**Review by BarVolunteering — COMMENTED** [review]
One more thing

**Review by BarVolunteering — APPROVED** [review]
Other than the question written above, 
Looks good!

File: anyway/views/news_flash/api.py (hunk 4 above)
Comment (ziv17) [reply anyway/views/news_flash/api.py:44]: Good catch!

**Pull request merged**